lib/Doctrine/Common/Proxy/ProxyGenerator.php

```diff
@@ -1108,10 +1110,21 @@ private function shouldProxiedMethodReturn(ReflectionMethod $method)
      * @return string
      */
     private function formatType(
-        ReflectionNamedType $type,
+        ReflectionType $type,
         ReflectionMethod $method,
         ?ReflectionParameter $parameter = null
     ) {
+        if ($type instanceof ReflectionUnionType) {
+            return implode('|', array_map(
+                function (ReflectionType $unionedType) use ($method, $parameter) {
+                    return $this->formatType($unionedType, $method, $parameter);
+                },
+                $type->getTypes()
```

Comment:
CodeCov says the lines are not covered, and it is correct: there is no PHP 8 job yet. I'll try to add one in a different PR.

Reply:
Now I rebased, it no longer complains :)

Reply:
I rebased, there no longer is a comment :+1: